src/main/c/Makefile

```diff
@@ -239,6 +239,13 @@ freebsd64: export platform := freebsd/x86_64
 freebsd64:
 	$(MAKE) -f natives.mk
 
+# Requires a FreeBSD host running on aarch64 (arm64) architecture.
+freebsdarm64v8: export CFLAGS += -I$(JAVA_HOME)/include/freebsd -m64
```

Comment:
You've removed the `-m64` linker flag; did you try without the compiler one, too?

Reply:
Sorry, I totally missed the compiler flag. I either need more caffeine or more vacation it seems ;-)